### PR TITLE
Made examples compatible with cargo-raze

### PR DESCRIPTION
**Pull request by UebelAndre**
Updated examples to be compatible with cargo raze which I then used to regenerate dependencies.

### CODE REVIEW

File: examples/ffi/ffi_examples.bzl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
```
Comment (damienmg) [inline examples/ffi/ffi_examples.bzl:1]: unless this is branched code.

```suggestion
# Copyright 2020 The Bazel Authors. All rights reserved.
```

File: examples/hello_runfiles/BUILD [hunk 2]
```diff
@@ -1,9 +1,20 @@
-package(default_visibility = ["//visibility:public"])
+# Copyright 2015 The Bazel Authors. All rights reserved.
```
Comment (damienmg) [inline examples/hello_runfiles/BUILD:1]:
```suggestion
# Copyright 2020 The Bazel Authors. All rights reserved.
```

**Review by damienmg — COMMENTED** [review]
I am not sure we want to do such a change. What would be the gain to do so?

**Comment by UebelAndre** [comment]
> I am not sure we want to do such a change. What would be the gain to do so?

I feel it makes the code more representative of what users would actually try to. It seemed like the examples might have used an older version of `raze` to generate the files that are currently committed. All I really did was add the necessary cargo files to be able to regenerate new ones. Can you talk more about why you might not want to make such a change?

**Comment by damienmg** [comment]
> > I am not sure we want to do such a change. What would be the gain to do so?
> 
> I feel it makes the code more representative of what users would actually try to. It seemed like the examples might have used an older version of `raze` to generate the files that are currently committed. All I really did was add the necessary cargo files to be able to regenerate new ones. Can you talk more about why you might not want to make such a change?

Although this is likely that the sys example was indeed make using raze (@illicitonion ?), the FFI ones were not. This adds more complexity for the ffi examples as the libc was handwritten and a simple use-case. I don't see value in putting it in raze format (it would actually remove an example of hand-writting a dependency).

**Comment by UebelAndre** [comment]
> Although this is likely that the sys example was indeed make using raze (@illicitonion ?), the FFI ones were not. This adds more complexity for the ffi examples as the libc was handwritten and a simple use-case. I don't see value in putting it in raze format (it would actually remove an example of hand-writting a dependency).

Hmm, I can remove the use of raze for the FFI example though I don't quite feel any information was lost in updating it to use raze nor do I feel it adds complexity (at least for users). I don't think it's valuable to have any example explicitly demonstrate that you can hand author BUILD files to be able to pull in dependencies. This is something that seems obvious given Bazel's repository rules. Though, that said, the FFI example uses just one dependency. I would care more if it were multiple. What do you think of the current state?

**Comment by illicitonion** [comment]
Yeah, the sys example was generated with Raze and then simplified a little.

I don't have strong feelings either way on this PR :)

**Comment by UebelAndre** [comment]
@damienmg after more thought I agree with you about the did examples 😀 I think it's better now

**Review by damienmg — APPROVED** [review]

**Comment by damienmg** [comment]
Nice thanks!

**Pull request merged**